source/reference/minio-mc/mc-replicate-diff.rst

```diff
@@ -0,0 +1,142 @@
+.. _minio-mc-replicate-diff:
+
+=====================
+``mc replicate diff``
+=====================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+.. mc:: mc replicate diff
+
+Description
+-----------
+
+.. start-mc-replicate-diff-desc
+
+The :mc:`mc replicate diff` shows a list of ``PUT`` or ``DELETE`` actions to objects and object versions that have not replicated to a remote bucket.
```

Comment:
This is technically accurate, but I think a little hard to parse. Maybe...

shows a list of unreplicated new or deleted objects for a given remote bucket ?